### PR TITLE
Feature/hn 10 create persistence layer for organizer model class

### PR DESCRIPTION
**Pull request by alexsandermeneses**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -74,6 +74,26 @@
 			<version>1.18.30</version>
 			<scope>provided</scope>
 		</dependency>
+
+		<dependency>
```
Comment (arnaldotecadm) [inline pom.xml:78]: why do we need this dependency? I was not expecting any dependency other than spring tests and junit/jupter

File: src/main/java/com/happeningnow/service/ServiceOrganizer.java [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+import java.util.Optional;
+import java.util.UUID;
+
+@AllArgsConstructor
+
+@Service
+public class ServiceOrganizer {
+
+    @Autowired
+    private final OrganizerRepository organizerRepository;
```
Comment (arnaldotecadm) [inline src/main/java/com/happeningnow/service/ServiceOrganizer.java:23]: please make this class receive this as a parameter, just like I showed you the other day.

File: src/main/java/com/happeningnow/service/ServiceOrganizer.java [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+    @Transactional
```
Comment (arnaldotecadm) [inline src/main/java/com/happeningnow/service/ServiceOrganizer.java:25]: Do not need to use transactional over here. Transactional is used when you have more than one data operation at the same time and you want them to be ACID

File: src/main/java/com/happeningnow/service/ServiceOrganizer.java [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+    public Organizer save(Organizer organizer){
+        return this.organizerRepository.save(organizer);
+    }
+
+    public Optional<Organizer> findById(UUID uuid){
+        return this.organizerRepository.findById(uuid);
+    }
+
+    public Page<Organizer> listOrganizer(@PageableDefault(sort = "name", direction = Sort.Direction.ASC, page = 1, size = 50) PageRequest pageable){ //tirei Pageable e coloquei PageRequest
```
Comment (arnaldotecadm) [inline src/main/java/com/happeningnow/service/ServiceOrganizer.java:34]: do not need to use the default over here, the default one will come from the controller. And delete the comment

File: src/main/java/com/happeningnow/service/ServiceOrganizer.java [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+        return organizerRepository.findAll(pageable);
+    }
+
+    public void deleteById(UUID uuid){ // ask / alteration void to Organizer // optional name altered delete to deleteById
```
Comment (arnaldotecadm) [inline src/main/java/com/happeningnow/service/ServiceOrganizer.java:38]: delete the comment

File: src/test/java/com/happeningnow/HappeningnowApplicationTests.java [hunk 6]
```diff
@@ -1,13 +1,8 @@
...
+
 @SpringBootTest
 class HappeningnowApplicationTests {
-
-	@Test
-	void contextLoads() {
-	}
-
-}
+}
```
Comment (arnaldotecadm) [inline src/test/java/com/happeningnow/HappeningnowApplicationTests.java:8]: Leave the class as it was, cause it ensures the context of the application is properly loaded. Avoiding the risk of forgetting to define any annotation or property

**Review by arnaldotecadm — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 1 above)
Comment (alexsandermeneses) [reply pom.xml:78]: Ok

File: src/main/java/com/happeningnow/service/ServiceOrganizer.java (hunk 5 above)
Comment (alexsandermeneses) [reply src/main/java/com/happeningnow/service/ServiceOrganizer.java:38]: Ok

File: src/main/java/com/happeningnow/service/ServiceOrganizer.java (hunk 4 above)
Comment (alexsandermeneses) [reply src/main/java/com/happeningnow/service/ServiceOrganizer.java:34]: Ok

File: src/main/java/com/happeningnow/service/ServiceOrganizer.java (hunk 3 above)
Comment (alexsandermeneses) [reply src/main/java/com/happeningnow/service/ServiceOrganizer.java:25]: Ok

File: src/test/java/com/happeningnow/HappeningnowApplicationTests.java (hunk 6 above)
Comment (alexsandermeneses) [reply src/test/java/com/happeningnow/HappeningnowApplicationTests.java:8]: Ok

File: src/main/java/com/happeningnow/service/ServiceOrganizer.java (hunk 2 above)
Comment (alexsandermeneses) [reply src/main/java/com/happeningnow/service/ServiceOrganizer.java:23]: I don´t remember this very well. I go study to make

**Review by arnaldotecadm — APPROVED** [review]
LGTM

**Pull request merged**